services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/pathverification/PathVerificationService.java

```diff
@@ -212,10 +215,23 @@ public Command call() throws Exception {
     protected List<OFAction> getDiscoveryActions(IOFSwitch sw, OFPort port) {
         // set actions
         List<OFAction> actions = new ArrayList<>();
+        actions.add(actionAddTxTimestamp(sw, 832));
```

Comment:
Is this offset in bits? Can we calculate it using our `discovery packet generator`? There is a risk that packet format will be altered and this "placeholder" will be moved.

From my point of view the best option is to put this 2 fields at the begin of payload, and shift LLDP on 16 bytes(2 timestamp fields) from the start of payload. I.e. 2 timestamp fields will precede LLDP and will be outside LLDP staff.

In this case we will guarantee constant timestamp placeholder's offsets. Also it is extremely simple to extract this fields and check their "existence". The only obstacle I see is the goal to keep "valid" LLDP packet (wrapped in UDP). Do we really need to keep valid LLD packet?